packages/edit-site/src/components/add-new-pattern/index.js

```diff
@@ -26,14 +26,13 @@ import {
 	TEMPLATE_PART_POST_TYPE,
 } from '../../utils/constants';
 
-const { useHistory, useLocation } = unlock( routerPrivateApis );
+const { useHistory } = unlock( routerPrivateApis );
 const { CreatePatternModal, useAddPatternCategory } = unlock(
 	editPatternsPrivateApis
 );
 
 export default function AddNewPattern() {
 	const history = useHistory();
-	const { params } = useLocation();
```

Comment:
I personally like the direct hook to retrieve the current route's params. Most routers have a similar hook. React Router has `useParams` for instance.